lib/cmd/fh3/appdata/import/start.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+/* globals i18n */
+
+var async = require('async');
+var path = require('path');
+var mime = require('mime');
+
+var fs = require('fs')
+  , fhreq = require('../../../../utils/request.js')
+  , readline = require('readline')
+  , common = require('../../../../common');
+
+function retrieveUploadMetadata(envId, appId, filePath, cb) {
+  var url = 'api/v2/appdata/' + envId + '/' + appId + '/data/import';
+
+  fs.stat(filePath, function (err, stat) {
+    if (err) {
+      return cb(err);
+    }
+
+    var fields = {
+      filename: path.basename(filePath),
+      filesize: stat["size"]
+    };
+    // Get the jobID
+    common.doApiCall(fhreq.getFeedHenryUrl(), url, fields, i18n._('Error importing data: '), function (err, jobData) {
+      if (err) {
+        return cb(err);
+      }
+
+      return cb(null, jobData);
+    });
+  });
+}
+
+function sendFile(filePath, jobData, cb) {
+  var uploadUrl = jobData.url;
+  var filename = path.basename(filePath);
+  var mimetype =  mime.lookup(filename);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'mimetype' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AUnusedVariable)